Script.js

```diff
@@ -215,4 +215,39 @@ workSection.addEventListener('click', (event) => {
     const getProjectId = event.target.parentElement.parentElement.id;
     popupWindow(getProjectId);
   }
+});
+
+//  Form Validation
+const form = document.querySelector('#contact_section form');
+const email = form.querySelector('#email');
+const error = form.querySelector('.error');
+const emailRegex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+const submitButton = form.querySelector('#form_submit');
+const senderMessage = form.querySelector('#msg');
+
+function setError(element, message) {
+  error.innerText = message;
+  element.classList.add('error-input');
+  error.classList.remove('hide');
+}
+
+function setSuccess(element) {
+  senderMessage.classList.add('sent');
+  element.classList.remove('error-input');
+  error.classList.add('hide');
+  submitButton.value = 'Sent';
+  submitButton.classList.add('success');
+}
+
+function validateEmail() {
+  const emailValue = email.value.trim();
+  if (!emailRegex.test(emailValue)) {
+    setError(email, 'Invalid email address');
+  } else {
+    setSuccess(email);
```

Comment:
- [ ] The email is being submitted even if it has an uppercase value, kindly fix this issue

![image](https://user-images.githubusercontent.com/81932478/202800236-db519ce7-f97f-4442-b509-d6ae089d35ab.png)

![image](https://user-images.githubusercontent.com/81932478/202800302-d1d0cd3f-2998-46f0-8282-bd7dbd4d2f2e.png)